docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -355,6 +355,29 @@ Each tablet replica generally requires 700 MiB of this memory.
 
 Default: `0` unless [`--use_memory_defaults_optimized_for_ysql`](#use-memory-defaults-optimized-for-ysql) is true.
 
+##### enforce_tablet_replica_limits
+
+Enables/disables the feature. No limits at all will be enforced if this is false.
+
+Default: false
+
+##### split_respects_tablet_replica_limits
+
+If set, tablets will not be split if the total number of tablet replicas in the cluster after the split would exceed the limit after the split.
+
+Default: false
+
+##### tablet_replicas_per_core_limit
+
+The number of tablet replicas that each core on a YB-TServer can support. This is used by both YB-Masters and YB-TServers.
+
+Default: `0` for no limit.
+
+##### tablet_replicas_per_gib_limit
+
+The number of tablet replicas that each GiB reserved by YB-TServer for tablet overheads can support. This is used by both YB-Masters and YB-TServers.
+
+Default: 1024 * (7/10) (corresponding to an overhead of roughly 700 KiB per tablet)
```

Comment:
I'm not sure these belong here. The text 

> Enables/disables the feature. No limits at all will be enforced if this is false.

doesn't make sense in this context - what feature? I'd perhaps move these to the `Sharding flags` section. Or possibly the "tablet splitting" section so the flag layout is parallel to the feature description layout, but tablet splitting doesn't quite feel right.

Also the text for the `enforce_tablet_replica_limits` flag should be something like - 
`Enables/disables blocking of requests which would bring the total number of tablets in the system over a limit. See <link> for more information.`

Reply:
Moving it to sharding flags.